### ISSUE TITLE
Change password design/contrast bug

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#1910]
**Describe the bug**

On the change password page, the text and logo doesn't have appropriate contrast.

**To Reproduce**
1. Go to MyProudCity login.
2. Select change password.

**Expected behavior**
Expect logo and text to be white.

**Screenshots**

![Screenshot 2021-10-19 at 13-23-46 Change your password](https://user-images.githubusercontent.com/1569463/138773772-bc976989-a5b3-4664-b57f-e8ea4072adb7.png)

**Desktop (please complete the following information):**
 - OS: Mac
 - Browser Firefox

## Comments

@aschmoe would be great to have an Auth0 clean-up session with you.

**Comment by lukefretwell** [comment]
Merging with #1917
